io.github.Archeb.opentrace.yml

```diff
@@ -1,6 +1,6 @@
 app-id: io.github.Archeb.opentrace
 runtime: org.gnome.Platform
-runtime-version: '42'
+runtime-version: '44'
```

Comment:
45 is the current one

Reply:
okay ill try that, currently working on migrating from extra-data..